package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "web3-onboard-monorepo",
-  "version": "2.14.0",
+  "version": "2.15.0",
```

```suggestion
  "version": "2.14.0",
```
We will bump this upon prod release